Add release notes for CCM 0.8.0 (#393)
chocolatey/docs: input/en-us/central-management/release-notes.md

* Please see https://github.com/chocolatey/chocolatey-licensed-issues/labels/CentralManagement
* Some issues may be held internally, please follow your support routes to learn more.

## 0.8.0 (February 28, 2022)

### Features

> - Ensure that a deployment step contains information about whether it contains sensitive variables in the script to run.

Reviewer: ```suggestion
- Ensure that a deployment step specifies whether it contains sensitive variables in the script that's to be run.
```
